android/app/src/main/res/values/dimens.xml

```diff
@@ -112,7 +112,7 @@
   <dimen name="nav_button_height">@dimen/primary_button_min_height</dimen>
   <dimen name="nav_icon_size">48dp</dimen>
   <dimen name="nav_bottom_gap">12dp</dimen>
-  <dimen name="nav_menu_height">70sp</dimen>
+  <dimen name="nav_menu_height">80sp</dimen>
```

Comment:
Nav panel now takes more map space, is it a really necessary change? Are there ways to fix issues without increasing the size?

Reply:
Since arabic characters take some extra spaces than normal english so we need to increase the size of nav in any case.

The other option could be to decrease the size of text little bit inside the bottom sheet nav then also it can get solved, but text might feel difficult to read for some people.

Reply:
Increasing the size manually will work until on some devices users set an even larger font that won't fit into the increased size. Right?

Android has `wrap_content` property that scales for the text size automatically (there are also other ways to scale). Why automatic scaling doesn't work here?